docs/MEETUP_TOPICS.md

```diff
@@ -0,0 +1,84 @@
+
+10. **Kubernetes Logging and Monitoring**  
+This topic will cover [basic logging architecture](https://kubernetes.io/docs/concepts/cluster-administration/logging/), [what to monitor](https://www.datadoghq.com/blog/monitoring-kubernetes-era/), which includes [cluster components](https://kubernetes.io/docs/tasks/debug-application-cluster/resource-usage-monitoring/). Reference: [kubectl logs](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#logs).
+
+11. **Intro to Helm**  
+Helm is a package manager for Kubernetes, like how `apt` is for `ubuntu`. This will not be in CKA exam but will be very useful, necessary, and practical to someone new to Kubernetes. Reference: [Helm docs](https://helm.sh/docs/intro/).
+
+12. **Extending Kubernetes**  
+This is the next step of Kubernetes, good for the last session of curriculum. It won't be in CKA exam but good for someone who has completed the entire core curriculum and looking to figure out what to learn next. This topic will cover [extending Kubernetes](https://kubernetes.io/docs/concepts/extend-kubernetes/extend-cluster/) with [kubectl plugins](https://kubernetes.io/docs/tasks/extend-kubectl/kubectl-plugins/), [Custom Resources](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/), what's [kubebuilder](https://github.com/kubernetes-sigs/kubebuilder) and [operator SDK](https://github.com/operator-framework/operator-sdk).
+
+
+#### Category #2: Coud Native Stack Exploration
+
+This category is as simple as *Intro to X*, replace X with any CNCF or [cloud native projects](http://l.cncf.io) that are open source. We won't go deep but go breadth to cover as many open source projects as we can. Prioritize the projects that have [graduated or are in incubating stages](https://www.cncf.io/projects/). Popular projects that at least should be covered:
+* Container runtime: [Docker](https://docker-curriculum.com/), [containerd](https://containerd.io/), [firecracker](https://github.com/firecracker-microvm/firecracker).
+* [Kubernetes](https://kubernetes.io/)
+* [Istio](https://istio.io/)
+* [Prometheus](https://prometheus.io/)
+* [Envoy](https://www.envoyproxy.io/)
+* [Open Policy Agent](https://www.openpolicyagent.org/)
+* [CoreDNS](https://coredns.io/)
+* [gRPC](https://grpc.io/)
+
+## Intermediate Track
+
+### Objective
+
+This track will cover practical usecases of Kubernetes and cloud native technologies adopted by companies, or can cover any intro to CNCF projects (like in Category #2 of Beginner Track) if a meetup decides to pick Category #1 already in the Beginner Track.
```

Comment:
`use cases` instead of `usescases`

Reply:
fixed in https://github.com/cloudnative-id/meetups/pull/33/commits/6a920f254edebd32fa1be3b8ec949a8015650052